src/status_im2/subs/contact_test.cljs

```diff
@@ -0,0 +1,183 @@
+(ns status-im2.subs.contact-test
+  (:require [cljs.test :refer [is testing]]
+            [re-frame.db :as rf-db]
+            [test-helpers.unit :as h]
+            status-im2.subs.contact
+            [utils.re-frame :as rf]))
+
+(def ^:private contacts-sample-data
```

Comment:
I would say privacy is irrelevant in test files because nothing should require a test file, except for the test runner.